storage/miner.go

```diff
@@ -179,12 +179,11 @@ func (m *Miner) Run(ctx context.Context) error {
 		adaptedAPI = NewSealingAPIAdapter(m.api)
 
 		// Instantiate a precommit policy.
-		cfg             = sealing.GetSealingConfigFunc(m.getSealConfig)
-		provingBoundary = md.PeriodStart % md.WPoStProvingPeriod
-		provingBuffer   = md.WPoStProvingPeriod * 2
+		cfg           = sealing.GetSealingConfigFunc(m.getSealConfig)
+		provingBuffer = md.WPoStProvingPeriod * 2
 
 		// TODO: Maybe we update this policy after actor upgrades?
```

Comment:
Can drop the comment. That comment was probably about removing the proving boundary (which we used to require).